timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -668,11 +668,9 @@ export default {
       const hiddenKeys = ["datetime", "timestamp_desc", "tag", "label", "comment", "tag", "label", "data_type", "domain", "hostname"]
       const regEx = /^[0-9]+-[0-9]+-[0-9]+[T][0-9]+[:][0-9]+[:][0-9]+/gm
       if (key.startsWith("__") || hiddenKeys.includes(key)) {
-        console.log(regEx.exec(value))
         return false
-      }
-      // Filter keys that contain ISO-8610 format 
-      else if (regEx.exec(value)) {
+      } 
+      else if (regEx.exec(value) || value == null) {
         return false
```

Comment:
Split the filters over separate statements in the following order:

if null
else keys starts with
else if hidden
else if regex timestamp

Reply:
Nevermind just combine the filters on a single if- statement.

```
if (value == null || key.startsWith("__") || hiddenKeys.includes(key) || regEx.exec(value)) {
  return false;
} else {
  return true;
}
```